### PR TITLE
Add more extensions

### PR DESCRIPTION
**Pull request by yanni4night**
Some OSX and MS office file extnames

### CODE REVIEW

**Comment by sindresorhus** [comment]
Don't change the indentation.

**Comment by SamVerschueren** [comment]
It's still wrong. It should be one tab, not spaces.

File: binary-extensions.json [hunk 1]
```diff
@@ -176,14 +217,22 @@
...
+	"xmind",
 	"xpi",
 	"xpm",
 	"xwd",
 	"xz",
 	"z",
 	"zip",
 	"zipx"
-]
+]
```
Comment (SamVerschueren) [inline binary-extensions.json]: Add newline

File: binary-extensions.json [hunk 2]
```diff
@@ -3,38 +3,49 @@
 	"3g2",
 	"3gp",
 	"7z",
+	"DS_Store",
```
Comment (SamVerschueren) [inline binary-extensions.json]: Put `DS_Store` back where it was.

**Comment by SamVerschueren** [comment]
I suggest to use the [EditorConfig](http://editorconfig.org/) plugin for your TextEditor.

**Comment by yanni4night** [comment]
Yeah, I created the json by nodejs and did not noticed that.

**Comment by sindresorhus** [comment]
You have added a lot of extensions, but I don't have time to verify them all. Either someone needs to do it or you need to provide info about them.

**Comment by yanni4night** [comment]
- [aiff](https://en.wikipedia.org/wiki/Audio_Interchange_File_Format): Audio Interchange File Format
- [alz](https://en.wikipedia.org/wiki/ALZip): ALZip is an archive and compression utility from ESTsoft for Microsoft Windows
- [ape](https://en.wikipedia.org/wiki/Monkey%27s_Audio): A filename extension used by Monkey's Audio.
- [arj](https://en.wikipedia.org/wiki/ARJ): ARJ (Archived by Robert Jung) is a software tool designed by Robert K. Jung for creating high-efficiency compressed file archives.
- [bh](http://fileinfo.com/extension/bh): File or group of files compressed with BlakHole compression
- [bzip2](http://fileinfo.com/extension/bzip2): Compressed archive containing one or more files that have been compressed using the bzip2 compression algorithm
- [csv](https://en.wikipedia.org/wiki/Comma-separated_values): In computing, a comma-separated values (CSV) file stores tabular data (numbers and text) in plain text.
- [cur](https://en.wikipedia.org/wiki/CUR): File format for cursor images from Microsoft
- [docm](https://en.wikipedia.org/wiki/Office_Open_XML): Office Open XML
- [dot](http://fileinfo.com/extension/dot): Template created by Microsoft Word
- [dotm](http://fileinfo.com/extension/dotm): Document template created by Microsoft Word
- [key](http://fileinfo.com/software/apple/keynote): Apple Keynote
- [lha](https://en.wikipedia.org/wiki/LHA_%28file_format%29): LHA is a freeware compression utility and associated file format.
- lzh: Same as `lha`
- [mht](https://en.wikipedia.org/wiki/MHTML): MHTML
- [midi](https://en.wikipedia.org/wiki/MIDI): Musical Instrument Digital Interface
- [mobi](https://en.wikipedia.org/wiki/Mobipocket): Mobipocket ebook format
- [numbers](http://fileinfo.com/extension/numbers): Apple Numbers
- [pages](http://fileinfo.com/extension/pages): Apple Pages
- [pot](http://fileinfo.com/extension/pot): Microsoft PowerPoint template files
- [potm](http://fileinfo.com/extension/potm): Macro-enabled presentation template created by Microsoft PowerPoint
- [potx](http://fileinfo.com/extension/potx): Presentation template created by PowerPoint
- [ppa](http://fileinfo.com/extension/ppa): Add-in file used by Microsoft PowerPoint
- [ppam](http://fileinfo.com/extension/ppam): Add-in file used by Microsoft PowerPoint
- [pps](http://fileinfo.com/extension/pps): Slide show created with Microsoft PowerPoint
- [ppsm](http://fileinfo.com/extension/ppsm): Macro-enabled slide show created by Microsoft PowerPoint
- [ppsx](http://fileinfo.com/extension/ppsx): Slide show created by Microsoft PowerPoint
- [ppt](http://fileinfo.com/extension/ppt): A PPT file is an editable slide show created with Microsoft PowerPoint often used for presentations or slideshows
- [pptm](http://fileinfo.com/extension/pptm): Macro-enabled presentation created by Microsoft PowerPoint
- [pptx](http://fileinfo.com/extension/pptx): Presentation file created by Microsoft PowerPoint
- [rmf](https://en.wikipedia.org/wiki/Rich_Music_Format): Rich Music Format
- [rmvb](https://en.wikipedia.org/wiki/RMVB): RealMedia Variable Bitrate
- [rtf](https://en.wikipedia.org/wiki/Rich_Text_Format): Rich Text Format
- [slk](https://en.wikipedia.org/wiki/SLK): SYmbolic LinK (SYLK) database/spreadsheet format
- [tbz](http://fileinfo.com/extension/tbz): Unix .TAR archive compressed with .BZIP compression
- [thmx](http://fileinfo.com/extension/thmx): Microsoft Office Theme file format
- [txz](http://fileinfo.com/extension/txz): Archive created with Tar and XZ utilities
- [udf](http://fileinfo.com/extension/udf): File created in the Universal Disk Format (UDF), a disc imaging format used for storing files on optical media
- [wim](http://fileinfo.com/extension/wim): File-based imaging format introduced with Windows Vista
- [xla](http://fileinfo.com/extension/xla): Adds modules, extra functions, and other tools to Microsoft Excel
- [xlam](http://fileinfo.com/extension/xlam): File used by Microsoft Excel, a program that allows users to create and edit spreadsheets; contains a macro-enabled add-in, which provides extra functionality and tools that may execute macros.
- [xlsb](http://fileinfo.com/extension/xlsb): Spreadsheet file created by Microsoft Excel
- [xlsm](http://fileinfo.com/extension/xlsm): Macro-enabled spreadsheet created by Microsoft Excel
- [xlt](http://fileinfo.com/extension/xlt): Template created by Microsoft Excel
- [xltm](http://fileinfo.com/extension/xltm): Template file created by Microsoft Excel
- [xltx](http://fileinfo.com/extension/xltx): Template file created by Microsoft Excel
- [xmind](http://fileinfo.com/extension/xmind): Workbook file created by XMind

---

The above all should be binary. Most of them are relative with Microsoft Office, OS X, media or archive.

**Comment by sindresorhus** [comment]
Thank you @yanni4night :)

**Pull request closed without merging**